multimodal/src/autogluon/multimodal/data/preprocess_dataframe.py

```diff
@@ -534,6 +551,13 @@ def transform_prediction(
             y_pred = np.squeeze(y_pred)
             # Convert nan to 0
             y_pred = np.nan_to_num(y_pred)
+        elif self.label_type == NER_ANNOTATION:
+            y_pred = self._label_generator.inverse_transform(y_pred)
+            if inverse_categorical:
+                # Return annotations and offsets
+                y_pred = y_pred[1]
+            else:
+                y_pred = y_pred[0]
```

Comment:
The `inverse_categorical` is for classification tasks. It seems not meaningful for the ner task (whether returning the offsets or not).

Reply:
NER is essentially a classification task. In the prediction stage, we need to tell a user which group an entity belongs

Reply:
Yeah, ner is a classification task. But I feel the `inverse_categorical ` may not applicable to ner. Here are some examples of what the label encoder does for `inverse_transform`. I think you have done the similar stuff in the ner `inverse_transform`.

Reply:
What the code does here is whether to return the offsets or not, which is unrelated to `inverse_categorical `.

Reply:
Oh, yeah. We need to get inversed label for both evaluation metric calculation and prediction output. So you are right it is used differently here. I used it to avoid adding another flag and changing the code in evaluate